app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -115,9 +115,7 @@ def create
         @collection.apply_depositor_metadata(current_user.user_key)
         add_members_to_collection unless batch.empty?
         @collection.collection_type_gid = params[:collection_type_gid] if params[:collection_type_gid]
```

Comment:
This could be rewritten as `@collection.collection_type_gid = params[:collection_type_gid].presence || default_collection_type_gid`, which really reduces the number of conditionals.